scripts/release/publish.sh

```diff
@@ -16,22 +55,27 @@ function checkout_master {
 
 function create_a_merge_commit_of_dev_into_master_and_tag_it {
     git merge dev --no-edit
-    git tag --annotate "v${PACKAGE_VERSION}" --message "v${PACKAGE_VERSION}"
+    git tag --annotate "v${NEW_PACKAGE_VERSION}" --message "v${NEW_PACKAGE_VERSION}"
 }
 
 function push_commit_and_tag_to_remote_master {
     git push origin master
-    git push origin "v${PACKAGE_VERSION}"
+    git push origin "v${NEW_PACKAGE_VERSION}"
 }
 
 function publish_release_on_sentry {
-    npx sentry-cli releases -o pix new -p pix-api "v${PACKAGE_VERSION}"
-    npx sentry-cli releases -o pix set-commits --commit "1024pix/pix@$( git rev-parse HEAD )" "v${PACKAGE_VERSION}"
-    npx sentry-cli releases -o pix finalize "v${PACKAGE_VERSION}"
+    npx sentry-cli releases -o pix new -p pix-api "v${NEW_PACKAGE_VERSION}"
+    npx sentry-cli releases -o pix set-commits --commit "1024pix/pix@$( git rev-parse HEAD )" "v${NEW_PACKAGE_VERSION}"
+    npx sentry-cli releases -o pix finalize "v${NEW_PACKAGE_VERSION}"
```

Comment:
En lisant (vite) la doc de Sentry j'ai l'impression que le `finalize` ne devrait être invoqué qu'au moment où la release passe effectivement en production (donc plus ici). Sinon, on prend le risque (entre autres) de réinitialiser côté Sentry les issues indiquées "fixed in next release" alors que le correctif n'est pas encore déployé.